inception/inception-search-core/src/main/java/de/tudarmstadt/ukp/inception/search/SearchService.java

```diff
@@ -108,6 +109,18 @@ long determineNumOfQueryResults(User aUser, Project aProject, String aQuery,
             SourceDocument aDocument, AnnotationLayer aAnnotationLayer,
             AnnotationFeature aAnnotationFeature)
         throws ExecutionException, IOException;
+    /*
+    public HashMap<String, HashMap<String, Double>> getProjectTextStatistics(User aUser,
+            Project aProject, String aStatistic, SourceDocument aDocument,
+            AnnotationLayer aAnnotationLayer, AnnotationFeature aAnnotationFeature,
+            String[] aPunctuationMarks)
+        throws IOException, ExecutionException;
+
+     */
+
+    public Map<String, Map<String, Object>> getProjectStatistics(User aUser, Project aProject, String aStatistic,
```

Comment:
I would create a class for this `Map<String, Map<String, Object>>` instead of using this nested type